src/pages/[...slug].tsx

```diff
@@ -158,19 +184,45 @@ const ListItem = (props: ChildOnlyProp) => (
   <chakra.li color="text300" {...props} />
 )
 
+const Img = (img: any) => {
+  // use router to get correct image relative path inside /public/content/ dynamically
+  const router = useRouter()
+  // TODO: update how `imgRelativePath` is computed for translated assets inside /translations, will depend on value of locale after setting up i18n
+  const imgRelativePath = `/content${router.asPath}/${img.src.slice(1)}`
```

Comment:
We could use `path.join` here.

There is a small bug in the generation of these paths. It is adding a `//` after the path.
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/6e32d9ba-af87-4489-9141-1d515129c86c)

Reply:
Nice, thanks, I've added a util in the last commit that fixes this using `path.join`